apps/web/src/app/(private)/evaluations/(root)/[evaluationUuid]/destroy/page.tsx

```diff
@@ -25,7 +25,7 @@ export default function DestroyEvaluation({
         !open && navigate.push(ROUTES.evaluations.root)
       }
       action={destroy}
-      submitStr={`Delete ${evaluation.name}`}
```

Comment:
delete evaluation al menos no?

Reply:
<img width="646" alt="image" src="https://github.com/user-attachments/assets/ff13d827-4b97-4103-a2b9-56c901112d0a">
Already clear with the title imo